src/app/legacy/containers/Navigation/index.jsx

```diff
@@ -55,6 +55,9 @@ const NavigationContainer = ({ propsForTopBarOJComponent }) => {
   const { blocks = [] } = propsForTopBarOJComponent || {};
   const { script, translations, navigation, service, dir } =
     use(ServiceContext);
+  const { enabled: isGlobalLanguageHomepageEnabled } = useToggle(
+    'globalLanguageHomepage',
+  );
```

Comment:
I would rename this toggle to more accurately describe what it does - sometimes this can be the name of the component `languageNavigation`,  `collapsibleNavigation` or `globalNavigation`?  

We need to remove the concept of Global Language Homepage from the Navigation, so that it's easier to roll out in future.

```suggestion
  const { enabled: isGlobalLanguageHomepageEnabled } = useToggle(
    'globalLanguageHomepage',
  );
```